src/coreComponents/mesh/generators/InternalWellGenerator.cpp

```diff
@@ -220,28 +223,45 @@ void InternalWellGenerator::constructPolylineNodeToSegmentMap()
     real64 const segmentLength = LvArray::tensorOps::l2Norm< 3 >( vSeg );
 
     // various checks and warnings on the segment and element length
-    GEOSX_THROW_IF( segmentLength < 1e-2,
-                    "Error in the topology of well " << getName() <<
-                    ": we detected a polyline segment measuring less than 1 cm",
+    GEOSX_THROW_IF( segmentLength < 1e-5,
```

Comment:
I'm afraid that (here and below) you're just hard coding other values that will fail in other contexts.
You should define default values but let the user change this if needed.

Reply:
I added an input parameter to let the user choose the value.